### ISSUE TITLE
My assignments shows all assessments

### ISSUE THREAD

**Issue by dirkroorda** [Dans-labs/dariah#2]
By Femmy: the `my assignments` button shows all self assessments, not just the ones assigned to me as a reviewer.

**Comment by dirkroorda** [comment]
Right. Confirmed. It seems that I had prepared the logic behind `my assignments` without finishing it properly.

The server call `mylist` asks for items that I own or that I am and editor of.

The server call `ourlist` asks for items that mention me in one or more other fields, to be specified in the data model under the key `ourFields`.

The data model for assignments specifies the fields `reviewerE` and `reviewerF` as `ourFields`. 

That means that asking the server for `ourlist` of the assessments, will get those that mention me as a reviewer.

This is half the story. When an assessment gets modified, and when that modification is in one of the `reviewerE` and `reviewerF` fields, this has implications for the state at the client:

If the table `assessment` is already on the state, and if it has a key `ourIds`, meaning that there has been a call before to ask for our assignments, then the contents of `ourIds` may have to be changed.

I have implemented `ourlist` on the server and I have written the logic for state updates in response to changes in the values of an `ourField`.

Not yet online, I'll do that in a minute, when all goes well ...
